redisinsight/ui/src/components/csrf/Csrf.spec.tsx

```diff
@@ -1,83 +1,86 @@
 import React from 'react'
+import { useDispatch, useSelector } from 'react-redux'
 
-import apiService, { setApiCsrfHeader } from 'uiSrc/services/apiService'
-import { render, screen, waitFor } from 'uiSrc/utils/test-utils'
-import Csrf from './Csrf'
+import { render, screen, initialStateDefault } from 'uiSrc/utils/test-utils'
+import Csrf from 'uiSrc/components/csrf/Csrf'
 
-jest.mock('uiSrc/services/apiService', () => ({
-  setApiCsrfHeader: jest.fn(),
-  get: jest.fn(() => ({ data: { token: 'csrf-token' } })),
+jest.mock('react-redux', () => ({
+  useDispatch: jest.fn(),
+  useSelector: jest.fn(),
 }))
```

Comment:
could you please mock 'uiSrc/slices/app/csrf', and check which actions we call.
You can find examples how we do this in other components